main.c

```diff
@@ -473,9 +473,14 @@ static void RenderScreen(SDL_Window *window, SDL_Renderer *renderer, SDL_Texture
   }
   uint64 t1 = SDL_GetPerformanceCounter();
   bool hq = RenderScreenWithPerf(pixels, pitch, g_ppu_render_flags);
-  if (g_display_perf)
-    RenderNumber(pixels + (pitch*2<<hq), pitch, g_curr_fps, hq);
-
+  if (g_display_perf) {
+    RenderNumber(pixels + (pitch * 2 << hq), pitch, g_curr_fps, hq);
+  }
+  if (g_config.display_perf_title) {
+    char title[] = "";

```

Comment:
This is a buffer overflow. I'd advise writing char title[sizeof(kTitle) + 20] instead and use snprintf

Reply:
Whoops, sorry. C is not my native language xP
Should be resolved, tho I hardcoded the value of 60 for the title buffer to reduce calculations.